app/controllers/ctc/portal/portal_controller.rb

```diff
@@ -37,6 +37,7 @@ def resubmit
         flash[:alert] = I18n.t("general.enable_javascript")
         return redirect_back(fallback_location: ctc_portal_edit_info_path)
       end
+      return redirect_back(fallback_location: ctc_portal_edit_info_path) unless @submission.tax_return.under_submission_limit?
```

Comment:
I know elsewhere in this controller action we `redirect_back`, but I personally prefer to specifically redirect to where we need to go.

e.g. `ApplicationController#redirect_to_intake_after_triage` has these lines:

```
  def redirect_to_intake_after_triage
    if Rails.env.production?
      redirect_to Questions::OverviewController.to_path_helper
    else
      redirect_to Questions::EnvironmentWarningController.to_path_helper
    end
  end
```

If you know where to go, why go "back" with a "fallback" when you can go there? :)